draft-ma-opsawg-ucl-acl.md

```diff
@@ -394,6 +394,113 @@ file="ietf-schedule@2023-01-19.yang"
 <CODE ENDS>
 ~~~~
 
+
+### Examples
+
+   Here are some delicated examples of period and recurrence formats defined as
+   YANG groupings. The data is provided in JSON.
+
+#### Period of Time
+
+   The period starting at 08:00:00 UTC, on January 1, 2023 and ending at 18:00:00 UTC
+   on December 31, 2025:
+
+~~~~
+{
+  "period-of-time": {
+    "explicit-start": "2023-01-01T08:00:00Z",
+    "explicit-end": "2025-12-01T18:00:00Z"
+  }
+}
+~~~~
+
+   The period starting at 08:00:00 UTC, on January 1, 2023 and lasting 15 days and
+   5 hours and 20 minutes:
+
+~~~~
+{
+  "period-of-time": {
+    "start": "2023-01-01T08:00:00Z",
+    "duration": "P15DT05:20:00"
+  }
+}
+~~~~
+
+   The period starting at 08:00:00 UTC, on January 1, 2023 and lasting 20 weeks:
+
+~~~~
+{
+  "period-of-time": {
+    "start": "2023-01-01T08:00:00Z",
+    "duration": "P20W"
+  }
+}
+~~~~
+
+#### Recurrence Rule
+
+  Every day in December
+
+~~~
+{
+  "recurrence": {
+    "freq": "daily",
+    "bymonth": "12"
+  }
+}
+~~~  
+
+   10 occurrences that occur every last Saturday of the month:
```

Comment:
```suggestion
   The following snippet can be used to indicate 10 occurrences that occur every last Saturday of the month:
```

Reply:
Isn't "direction" a leaf-list? Or I'm not reading the latest version of the yang module?

Reply:
Sorry, the direction is indeed a leaf-list, I will fix the json snippet.